tools/cldr-apps/src/main/java/org/unicode/cldr/web/ReportsDB.java

```diff
@@ -67,9 +67,8 @@ public ReportStatus getReportStatus(Integer user, CLDRLocale locale) {
             PreparedStatement ps = DBUtils.prepareStatementWithArgsFRO(conn,
                 String.format("SELECT report, completed, acceptable FROM %s WHERE submitter=? AND locale=?", table),
                 user, locale.getBaseName());
-            ResultSet rs = ps.executeQuery();
-        ) {
-            while(rs.next()) {
+            ResultSet rs = ps.executeQuery();) {
+            while (rs.next()) {
                 final String report = rs.getString("report");
                 final Boolean completed = rs.getBoolean("completed");
                 final Boolean acceptable = rs.getBoolean("acceptable");
```

Comment:
future improvement: constants instead of string literals like "acceptable", for java/js/vue -- more maintanable, since these are common words that also occur in irrelevant contexts